pallets/phala-world/src/traits.rs

```diff
@@ -120,17 +150,28 @@ pub struct FoodInfo<BoundedOriginOfShellsFed> {
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 pub struct ShellPartInfo<BoundedString, BoundedSubParts> {
 	pub shell_part: PartInfo<BoundedString>,
+	// pub shell_part: BoundedPartInfo,
```

Comment:
We can remove this, right?

Reply:
Yes of cause, forget to do some clean up.